examples/provider/bid/audience-network/README.md

```diff
@@ -0,0 +1,51 @@
+# audience-network-bid-delegate
+
+BidProvider for Audience Network
+
+## Usage
+
+1. Create a new App at https://developers.facebook.com/apps
+2. Add the Audience Network product to it
+3. Create a new Placement to generate your `placementId`
+4. To test, ensure the `User-Agent` request header represents a mobile device
+
```

Comment:
We only bid on approved publishers. The readme can mention a publisher should speak to their account manager to start the approval process.